src/v2/Apps/Artist/Routes/WorksForSale/Utils/getWorksForSaleRouteVariables.ts

```diff
@@ -1,11 +1,17 @@
 import { getDefaultSortValueByVariant } from "v2/Utils/merchandisingTrial"
 import { getInitialFilterState } from "v2/Components/ArtworkFilter/Utils/getInitialFilterState"
 
-export function getWorksForSaleRouteVariables({ artistID }, props) {
+export function getWorksForSaleRouteVariables(
+  { artistID },
+  { location, context }
+) {
+  const { featureFlags = {} } = context
+
   // FIXME: The initial render includes `location` in props, but subsequent
   // renders (such as tabbing back to this route in your browser) will not.
-  const initialFilterState = getInitialFilterState(props.location?.query ?? {})
+  const initialFilterState = getInitialFilterState(location?.query ?? {})
   const defaultSort = getDefaultSortValueByVariant(artistID)
+  const newPriceFilterFlag = featureFlags["new_price_filter"]
```

Comment:
> Switching from the ENV var to this is because we're using Unleash now for this, yes?

I thought it would be more convenient to use Unleash + through `getEnv` I was unable to get the value of `ENABLE_NEW_PRICE_FILTER` during server redering.